### PR TITLE
like_post

### PR DESCRIPTION
**Pull request by vutuananh140697**
like post
checed
https://edu-redmine.sun-asterisk.vn/issues/15227
Screenshot from 2019-07-26 11-41-55

### CODE REVIEW

**Comment by vutuananh140697** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -580,6 +580,11 @@ a.uitext, .uitext a {
 
 // search
 
+#searchDiv {
+  height:34px;
+  width:196px;
```
Comment (datcnt-0874) [inline app/assets/stylesheets/custom.scss]:
```suggestion
  width: 196px;
```

File: app/assets/stylesheets/custom.scss [hunk 2]
```diff
@@ -580,6 +580,11 @@ a.uitext, .uitext a {
 
 // search
 
+#searchDiv {
+  height:34px;
```
Comment (datcnt-0874) [inline app/assets/stylesheets/custom.scss]:
```suggestion
  height: 34px;
```

**Comment by vutuananh140697** [comment]
ready

**Pull request merged**